KnownMimeTypes.cs

```diff
@@ -5149,1825 +4326,80 @@ public static class KnownMimeTypes
xt_jpeg:				case _ext_jpg:				case _ext_jpe:					return Jpeg;				case _ext_ktx:					return Ktx;				case _ext_png:					return Png;				case _ext_btif:					return Btif;				case _ext_sgi:					return Sgi;				case _ext_svg:				case _ext_svgz:					return Svg;				case _ext_tiff:				case _ext_tif:					return Tiff;				case _ext_psd:					return Psd;				case _ext_uvi:				case _ext_uvvi:				case _ext_uvg:				case _ext_uvvg:					return Uvi;				case _ext_djvu:				case _ext_djv:					return Djvu;				case _ext_sub:					return Sub;				case _ext_dwg:					return Dwg;				case _ext_dxf:					return Dxf;				case _ext_fbs:					return Fbs;				case _ext_fpx:					return Fpx;				case _ext_fst:					return Fst;				case _ext_mmr:					return Mmr;				case _ext_rlc:					return Rlc;				case _ext_mdi:					return Mdi;				case _ext_wdp:					return Wdp;				case _ext_npx:					return Npx;				case _ext_wbmp:					return Wbmp;				case _ext_xif:					return Xif;				case _ext_webp:					return Webp;				case _ext_3ds:					return _3ds;				case _ext_ras:					return Ras;				case _ext_cmx:					return Cmx;				case _ext_fh:				case _ext_fhc:				case _ext_fh4:				case _ext_fh5:				case _ext_fh7:					return Fh;				case _ext_ico:				case _ext_cur:					return Ico;				case _ext_sid:					return Sid;				case _ext_pcx:					return Pcx;				case _ext_pic:				case _ext_pct:					return Pic;				case _ext_pnm:					return Pnm;				case _ext_pbm:					return Pbm;				case _ext_pgm:					return Pgm;				case _ext_ppm:					return Ppm;				case _ext_rgb:					return Rgb;				case _ext_tga:					return Tga;				case _ext_xbm:					return Xbm;				case _ext_xpm:					return Xpm;				case _ext_xwd:					return Xwd;				case _ext_eml:				case _ext_mime:					return Eml;				case _ext_igs:				case _ext_iges:					return Igs;				case _ext_msh:				case _ext_mesh:				case _ext_silo:					return Msh;				case _ext_dae:					return Dae;				case _ext_dwf:					return Dwf;				case _ext_gdl:					return Gdl;				case _ext_gtw:					return Gtw;				case _ext_mts:					return Mts;				case _ext_vtu:					return Vtu;				case _ext_wrl:				case _ext_vrml:					return Wrl;				case _ext_x3db:				case _ext_x3dbz:					return X3db;				case _ext_x3dv:				case _ext_x3dvz:					return X3dv;				case _ext_x3d:				case _ext_x3dz:					return X3d;				case _ext_appcache:				case _ext_manifest:					return Appcache;				case _ext_ics:				case _ext_ifb:					return Ics;				case _ext_css:					return Css;				case _ext_csv:					return Csv;				case _ext_html:				case _ext_htm:				case _ext_shtml:					return Html;				case _ext_n3:					return N3;				case _ext_txt:				case _ext_text:				case _ext_conf:				case _ext_def:				case _ext_list:				case _ext_log:				case _ext_in:				case _ext_ini:					return Txt;				case _ext_dsc:					return Dsc;				case _ext_rtx:					return Rtx;				case _ext_sgml:				case _ext_sgm:					return Sgml;				case _ext_tsv:					return Tsv;				case _ext_t:				case _ext_tr:				case _ext_roff:				case _ext_man:				case _ext_me:				case _ext_ms:					return T;				case _ext_ttl:				
```

Comment:
Looks like some TT whitespace changes causes this switch case statement to be one giant line. Can you fix this?

Reply:
Saw that too... will do!